doc/help/kdb-cp.md

```diff
@@ -11,7 +11,7 @@ You can copy keys to another directory within the database or even below another
 Note that you cannot copy a key below itself.
 
 Where `source` is the path of the key(s) you want to copy and `dest` is the path where you would like to copy the key(s) to.
-Note that when using the `-r` flag, `source` as well as all of the keys below it will be copied.
```

Comment:
This is a similar case as before. It seems that this is the correct or more sensible form already.

Reply:
Thank you for the feedback!
Please see my comment for your other similar suggestion.